extension/src/pipeline/quickPick.ts

```diff
@@ -79,16 +75,16 @@ const pickFields = async (
     )
   }
 
-  const xValue = await quickPickValue(items, {
+  const xValues = (await quickPickUserOrderedValues(items, {
     title: Title.SELECT_PLOT_X_METRIC
-  })
+  })) as { file: string; key: string }[] | undefined
 
-  if (!xValue) {
+  if (!xValues || xValues.length === 0) {
     return
   }
 
-  const yValues = (await quickPickManyValues(
-    items.filter(item => !isEqual(item.value, xValue)),
+  const yValues = (await quickPickUserOrderedValues(
+    items.filter(item => xValues.every(val => !isEqual(val, item.value))),
```

Comment:
Is there a simpler way to filter out items that are in `xValues`? Right now we've basically got a nested loop 🤔